src/coreclr/interop/interoplib.cpp

```diff
@@ -73,8 +73,6 @@ namespace InteropLib
             if (mow == nullptr)
                 return E_INVALIDARG;
 
-            (void)mow->AddRef();
```

Comment:
Why can we delete this AddRef?

Reply:
This is a bug.

Reply:
I found it while testing the newly exposed `Unwrap` API. I am working with the C#/WinRT team to determine if it is needed for .NET 5. Likely it will be needed.